### PR TITLE
Update code to Swift 3

### PR DESCRIPTION
**Pull request by alexsinger**

### CODE REVIEW

File: Cannonball/SignInViewController.swift [hunk 1]
```diff
@@ -65,51 +65,51 @@ class SignInViewController: UIViewController, UIAlertViewDelegate {
...
+        configuration?.appearance = DGTAppearance()
+        configuration?.appearance.backgroundColor = UIColor.cannonballBeigeColor()
+        configuration?.appearance.accentColor = UIColor.cannonballGreenColor()
 
         // Start the Digits authentication flow with the custom appearance.
-        Digits.sharedInstance().authenticateWithViewController(nil, configuration:configuration) { (session, error) in
+        Digits.sharedInstance().authenticate(with: nil, configuration:configuration!) { (session, error) in
             if session != nil {
                 // Navigate to the main app screen to select a theme.
                 self.navigateToMainAppScreen()
```
Comment (danasilver) [inline Cannonball/SignInViewController.swift]: Should wrap this in `DispatchQueue.main.async { }` to present on the main thread so next screen is shown.

```
DispatchQueue.main.async {
    // Navigate to the main app screen to select a theme.
    self.navigateToMainAppScreen()
}
```

File: Cannonball/ThemeChooserViewController.swift [hunk 2]
```diff
@@ -87,19 +111,19 @@ class ThemeChooserViewController: UITableViewController {
...
-        navigationController?.navigationBar.translucent = true
+        navigationController?.navigationBar.isTranslucent = true
     }
 
-    override func viewWillDisappear(animated: Bool) {
+    override func viewWillDisappear(_ animated: Bool) {
         super.viewWillDisappear(animated)
 
         // Move the logo view off screen if a new controller was pushed.
         if navigationController?.viewControllers.count > 1 {
```
Comment (danasilver) [inline Cannonball/ThemeChooserViewController.swift]: Change to `if let vcCount = navigationController?.viewControllers.count, vcCount > 1 {` so we can remove the `FIXME` for comparison operators with optionals.

File: Cannonball/ThemeChooserViewController.swift [hunk 3]
```diff
@@ -17,6 +17,30 @@
 import UIKit
 import QuartzCore
 import Crashlytics
+// FIXME: comparison operators with optionals were removed from the Swift Standard Libary.
```
Comment (danasilver) [inline Cannonball/ThemeChooserViewController.swift]: Can remove both of these `FIXME`s after the change below.

**Review by danasilver — CHANGES_REQUESTED** [review]

**Comment by alexsinger** [comment]
@danasilver I made your suggested changed. Let me know how it looks. Thanks!

**Review by danasilver — APPROVED** [review]

**Comment by danasilver** [comment]
Looks great! Ship it! 💯

**Pull request merged**